src/app/controllers/OverviewController.ts

```diff
@@ -190,8 +193,18 @@ export default class OverviewController {
 			}
 		}
 
-		const bestRank = findLowest(validators, (cur) => cur.data.rank7d)
-		const worstRank = findHighest(validators, (cur) => cur.data.rank7d)
+		let bestRank = 0
+		let bestTopX = 0
+		let worstRank = 0
+		let worstTopX = 0
```

Comment:
I would personally rename these to `bestPercentage` and `worstPercentage`. The current names led me to believe they have something so with charts and x-axes.

Reply:
Fixed (to `bestTopPercentage` and `worstTopPercentage`).